todolist/app/screens/BookDetail/index.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+
+import Detail from './layout';
+
+class BooKDetail extends Component {
+  static navigationOptions = ({ navigation }) => {
+    const item = navigation.getParam('item',{});
+    return {
+      title: item.title
+    }
+  };
+  render() {
+    const item = this.props.navigation.getParam('item',{});
```

Comment:
`this.props.navigation.getParam('item');` i don't think that you need the `{}` part